test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
```

Comment:
Not sure how we should go about testing the model, but [this post](https://semaphoreci.com/community/tutorials/how-to-test-rails-models-with-rspec) might be something to look into.